src/playercards/CardSearch.ttslua

```diff
@@ -116,18 +118,19 @@ function search()
     table.sort(cardList, function(k1, k2) return spawnAll == (k1.data.Nickname > k2.data.Nickname) end)
 
     local rot = self.getRotation()
-    local pos = self.positionToWorld(Vector(0, 0, -0.225))
-    for _, card in ipairs(cardList) do
-        pos[2] = pos[2] + 0.1
-        spawnObjectData({
-            data     = card.data,
-            position = pos,
-            rotation = rot
-        })
-        if not spawnAll then return end
-        if i == 100 then
-            printToAll("Only first 100 results of " .. #cardList .. " total results spawned.", "Yellow")
-            return
-        end
-    end
+    local pos = self.positionToWorld(Vector(0, 2, -0.225))
+    Spawner.spawnCards(cardList, pos, rot, true)
+    -- for _, card in ipairs(cardList) do
+    --     pos[2] = pos[2] + 0.1
+    --     spawnObjectData({
+    --         data     = card.data,
+    --         position = pos,
+    --         rotation = rot
+    --     })
+    --     if not spawnAll then return end
+    --     if i == 100 then
+    --         printToAll("Only first 100 results of " .. #cardList .. " total results spawned.", "Yellow")
+    --         return
+    --     end
+    -- end
```

Comment:
should probably get rid of the commented-out code (because git handles version control for us)